packages/@aws-cdk/aws-ecs-patterns/lib/ecs/load-balanced-ecs-service.ts

```diff
@@ -43,32 +58,44 @@ export class LoadBalancedEc2Service extends LoadBalancedServiceBase {
    */
   public readonly service: ecs.Ec2Service;
 
+  /**
+   * Constructs a new instance of the LoadBalancedEc2Service class.
+   */
```

Comment:
I don't see comments for public constructors on other constructs -- is this necessary?

Reply:
We've done this in the aws-ecs module, we'll need to do this when we get to documenting this when it's no longer experimental.

Reply:
Added comments to all of the constructs